Add `newInstance` option for macOS (#253)
sindresorhus/open: readme.md

Do not bring the app to the foreground.

##### newInstance <sup>(macOS only)</sup>

Type: `boolean`\
Default: `false`

> Open a new instance of the application(s) even if one is already running.

Reviewer: ```suggestion
Open a new instance of the app even it's already running.
```

> On other platforms, this is always to open a new instance.

Reviewer: This needs to be better written.

Author: Updated to `For other platforms, this value is always to be true.`
is this better?
